eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -54,14 +54,6 @@ jobs:
 
   steps:
 
-  - task: AzureKeyVault@2
```

Comment:
Don't need this anymore since we are not retrieving the con string from KV anymore.